howso/analysis.amlg

```diff
@@ -78,6 +78,10 @@
 			;list of features whose values to use to rebalance case weighting of the data and to store into weight_feature
 			rebalance_features (null)
 
+			;{type "boolean"}
+			;when true, the default, will compute and use a sparse deviation matrix for each nominal feature in all similarity queries.
+			; This incurs a small penalty in speed, but can improve model performance in data with unbalanced nominal features.
```

Comment:
"This incurs a penalty in speed, but can improve model performance in data with unbalanced nominal features."